node/narwhal/src/helpers/storage.rs

```diff
@@ -296,7 +296,10 @@ impl<N: Network> Storage<N> {
         self.certificates.write().remove(&certificate_id);
         // Remove the batch ID.
         self.batch_ids.write().remove(&batch_id);
-        // TODO (howardwu): Remove the transmissions.
+        // Remove the transmissions.
```

Comment:
This case is not so easy.

We can only remove the transmission if this certificate is the **last** instance that references the transmission ID.

This was why it was initially left as a TODO to resolve.

Reply:
Oh, then we'll just keep this as a draft for now.